geomstats/geometry/fiber_bundle.py

```diff
@@ -153,11 +271,14 @@ class FiberBundle(ABC):
     def __init__(self, total_space, aligner=None):
         self._total_space = total_space
         if aligner is True:
-            aligner = (
-                DistanceMinimizationBasedAligner(total_space)
-                if not gs.__name__.endswith("numpy")
-                else None
-            )
+            if isinstance(total_space.group_action, tuple):
+                aligner = AlternateAlignmentAlgorithm(total_space)
```

Comment:
NIT: AlternateAlignmentAlgorithm --> AlternatingAligner (shorter, and closer to the other names we use that end with "Aligner" and not "Alignment". The "Algorithm" part is not needed.)